@shared/api/helpers/getIconUrlFromIssuer.ts

```diff
@@ -56,7 +58,9 @@ export const getIconUrlFromIssuer = async ({
   try {
     /* Otherwise, 1. load their account from the API */
     const { networkUrl } = networkDetails;
-    const server = new StellarSdk.Server(networkUrl);
+    const server = new StellarSdk.Server(networkUrl, {
+      allowHttp: getIsAllowHttp(networkDetails.networkUrl),
+    });
```

Comment:
this PR contains a lot of this type of change: we need to `allowHttp` for users running Horizon locally via Stellar Quickstart